### PR TITLE
Fix non-unique font names

### PR DESCRIPTION
**Pull request by chucker**
See #452.

This is only a workaround. However, given that `FindFont` compares by name, we probably need to do something like this. (We might be able to better filter/order which fonts are "valid" or most likely to be the one referenced in the SVG?)

### CODE REVIEW

**Comment by chucker** [comment]
Hmmm, maybe I should have made the pull request to `netcore` instead. Is anything in particular left to do for the `netcore` branch to be closed?

File: Source/Document Structure/SvgGroup.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ protected override void Render(ISvgRenderer renderer)
         }
 
         /// <summary>
-        /// Gets the <see cref="GraphicsPath"/> for this element.
+        /// Gets the <see cref="System.Drawing.Drawing2D.GraphicsPath"/> for this element.
```
Comment (H1Gdev) [inline Source/Document Structure/SvgGroup.cs]: Is this change unnecessary ?

**Comment by chucker** [comment]
(Will do when I get around to it; maybe this weekend)

**Comment by mrbean-bremen** [comment]
Sure, thanks!

**Comment by mrbean-bremen** [comment]
@chucker - if you don't mind, I will just incorporate your fix manually to have it in for the next release, as it is actually a small change.

**Comment by chucker** [comment]
Sure, that’s fine!

**Comment by mrbean-bremen** [comment]
Thanks!

**Pull request closed without merging**